### ISSUE TITLE
ContainerService WindowsProfile adminPassword validation

### ISSUE THREAD

**Issue by sozercan** [Azure/azure-sdk-for-go#986]
Getting error during adminPassword validation:
```
* azurerm_container_service.test: containerservice.ContainerServicesClient#CreateOrUpdate: Invalid input: autorest/validation: validation failed: parameter=parameters.Properties.WindowsProfile.AdminPassword constraint=Pattern value="CHANGE_me1234!" details: error parsing regexp: invalid or unsupported Perl syntax: `(?=`
```

https://github.com/Azure/azure-sdk-for-go/blob/master/services/containerservice/mgmt/2017-09-30/containerservice/containerservices.go#L69
I don't believe Go supports lookaheads in regex, looks like invalid syntax

@marstr @mcardosos

**Comment by jhendrixMSFT** [comment]
It looks like this is the only service that depends on lookaheads which is good news.
There is package [regexp2](https://github.com/dlclark/regexp2) which is a port of the regex engine from .NET Core which does support lookaheads however I question taking a dependency on an entirely new regex engine to satisfy one package.  The other option would be to omit regex validation in such cases.

**Comment by jhendrixMSFT** [comment]
This has been fixed in the generator and will show up in the v14 release.

**Comment by jhendrixMSFT** [comment]
Fixed in v14.0.0.
